irods_testing_environment/irods_setup.py

```diff
@@ -150,9 +159,13 @@ def setup(self,
         negotiation_key -- secret key used in server-to-server communication
         control_plane_key -- secret key shared by all servers
         admin_password -- password for the iRODS administration account
+        provides_local_storage -- indicates whether the server should provide storage
+        resource_name -- name used to identify the local storage
         vault_directory -- storage location of the default unixfilesystem resource created
                            during installation
         """
+        self.irods_version = irods_version
```

Comment:
Should probably throw an exception if `irods_version` is not a tuple of integers.

Thoughts?

Reply:
the validator will catch that on startup, no?

Reply:
This is internal to the testing environment. If it is not of the correct type, the testing environment will fail to stand things up.

I'm not sure what would happen if the type is a tuple, but the tuple elements make no sense.

I feel this isn't a major concern given the fact that the code that retrieves the version information is hard-coded to return a tuple based on the information defined in the version.json files.

Reply:
right, and we're the only ones right now running the testing environment.

Reply:
That's right.

Reply:
Eh, I think it's okay for now. Feel free to add the check if you'd like. I'm of the opinion that this whole file needs to be reworked eventually :p

Reply:
Ok. I'll leave it as is.